route/build/src/water_balance.f90

```diff
@@ -200,7 +201,7 @@ SUBROUTINE comp_global_wb(ixRoute, verbose, ierr, message)
     if (abs(wb_error) > 1._dp) then ! tolerance is 1 [m3]
       write(iulog,'(A,1PG15.7,1X,A)') ' WARNING: global WB error [m3] = ', wb_error, '> 1.0 [m3]'
     end if
-    flush(iulog)
+    call shr_sys_flush(iulog)
```

Comment:
shr_sys_flush

Reply:
Talking with @nmizukami maybe the best solution would be to remove this flush statement.

Reply:
I'm changing all the shr_sys_flush calls to the 2003 FORTRAN flush keyword.